packages/nc-gui/components/project/spreadsheet/components/editableCell/dateTimePickerCell.vue

```diff
@@ -42,7 +42,11 @@ export default {
           .format('YYYY-MM-DD HH:mm')
       },
       set(val) {
-        this.$emit('input', val && dayjs(val).format('YYYY-MM-DD HH:mm:ssZ'))
```

Comment:
Can you share more information like in which version it's failed and datatype you had used?

Reply:
![mysqlerror](https://user-images.githubusercontent.com/59797957/157045144-aaa6f1b2-a034-4906-9815-7faf7288483e.png)
Node: **v16.14.0**
Arch: **x64**
Platform: **win32**
Docker: **false**
Database: **mysql2**
ProjectOnRootDB: **false**
RootDB: **sqlite3**
PackageVersion: **0.84.14**

When I try to pick a datetime column on a MySQL table it gives this error. I didn't want to merge this into a feature but it showed up while I try to test the feature so I ended up with this solution.

Also I tried datetimes with timezones on different versions of MySQL. You can check this fiddle, which gives same error: https://www.db-fiddle.com/f/itVMsnmnr6LpB5mn4UimRp/0

Reply:
Thank you for the effort @mertmit . Really appreciate it. Possible to get in touch via n _AT_ nocodb.com